packages/tools/generator-pie-component/src/app/templates/pie-placeholder.__stories__.ts

```diff
@@ -15,9 +15,9 @@ const defaultArgs: <%= componentName %>Props = {};
 const <%= componentNameCamelCase %>StoryMeta: <%= componentName %>StoryMeta = {
     title: '<%= displayName %>',
     component: 'pie-<%= fileName %>',
-    argTypes: {
-    <% if (needsRTL) { %>...i18nArgTypes,<% } %>
-    },
+    argTypes: {<% if (needsRTL) { %>
```

Comment:
Previous the generated text had unnecessary line breaks when `needsRTL` was false, now it's `{}`.